api/tests/test_multilingual_support.py

```
@@ -201,26 +201,63 @@ async def test_detects_spanish(self, sample_spanish_query, mock_llm_provider):
         assert confidence >= 0.8
 
+    @pytest.mark.asyncio
+    async def test_detects_language_with_invoke_fallback(self, sample_german_query):
+        """Detector should support LLM wrappers that expose invoke()."""
+        from app.services.translation.language_detector import LanguageDetector
+
+        class InvokeOnlyLLM:
+            def __init__(self):
+                self.invoke_calls = 0
+
+            def invoke(self, prompt: str):
```

Ruff (0.15.2)

[warning] Unused method argument: `prompt`

(ARG002)